waltid-cli/README.md

```diff
@@ -13,34 +13,36 @@ Manage keys, DIDs, issue Verifiable Credentials, and verify them using the WaltI
+| Command                                                                                                                                                                                             | What it does                                                                                                            |
+|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------| 
+| `waltid -h`                                                                                                                                                                                         | Print usage message                                                                                                     |
+| `waltid --help`                                                                                                                                                                                     | Print WaltId CLI usage message                                                                                          |
+| `waltid key -h`                                                                                                                                                                                     | Print WaltId CLI key command usage message                                                                              |
+| `waltid key generate -h`                                                                                                                                                                            | Print WaltId CLI key generate command usage message                                                                     |
+| `waltid key convert -h`                                                                                                                                                                             | Print WaltId CLI key generate command usage message                                                                     |
+| `waltid key generate`                                                                                                                                                                               | Generates a cryptographic key of type Ed25519                                                                           |
+| `waltid key generate -tsecp256k1`                                                                                                                                                                   | Generates a cryptographic key of type secp256k1                                                                         |
```

Comment:
maybe make it same as the other generate key command ex : `waltid key generate --keyType=secp256k1`